.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+{"version":1,"defects":{"Varspool\\JobAdder\\ClientTest::testConstructor":4,"Varspool\\JobAdder\\ClientTest::testRealClient":5},"times":{"Varspool\\JobAdder\\ClientTest::testConstructor":0.029,"Varspool\\JobAdder\\ClientTest::testRealClient":0.297}}
```

Comment:
Looks like you didn't mean to commit this file.

Reply:
Yup. Gone now.